subprojects/enterprise/src/main/java/org/gradle/internal/enterprise/impl/DefaultGradleEnterprisePluginCheckInService.java

```diff
@@ -55,20 +57,29 @@ public DefaultGradleEnterprisePluginCheckInService(
         MINIMUM_SUPPORTED_PLUGIN_VERSION_FOR_CONFIGURATION_CACHING.getMajor(),
         MINIMUM_SUPPORTED_PLUGIN_VERSION_FOR_CONFIGURATION_CACHING.getMinor());
 
+    public static final VersionNumber MINIMUM_SUPPORTED_PLUGIN_VERSION_FOR_ISOLATED_PROJECTS = VersionNumber.version(3, 15);
+    public static final String UNSUPPORTED_PLUGIN_DUE_TO_ISOLATED_PROJECTS_MESSAGE = "The Gradle Enterprise plugin has been disabled as it is incompatible with isolated projects feature";
```

Comment:
❌Based on the input from @bamboo, we should call the feature "Project Isolation" instead of isolated projects. We should update the wording here to address that. Also, for all new methods/variables we should use "Project Isolation" naming as well.

One more thing is to drop the "The" in front of the message. I am not a native speaker, but this is what we settled on when working on the GE plugin deprecation message:
https://github.com/gradle/gradle/pull/25255#discussion_r1233863937

Reply:
`Isolated projects` vs `project isolation`
Good point, I've named this in such manner to be consistent with underlying `StartParameterBuildOption.IsolatedProjects`.